src/domains/agents/communication/proactive_agent.py

```
@@ -346,39 +454,69 @@ async def _generate_proactive_message(self, message_type: str, belief_content: D
+            else:
+                # Vary the fallback response to avoid repetition
+                import random
+                fallbacks = [
+                    f"Let me help you with that, {user_name}. What specific aspect would you like me to focus on?",
+                    f"I'm here to support your business operations, {user_name}. Could you tell me more about what you need?",
+                    f"That's something I can definitely assist with, {user_name}. What would be the most helpful approach?",
+                    f"I understand, {user_name}. Let me see how I can best support you with this business need."
+                ]
+                return random.choice(fallbacks)
+    
+    async def _show_typing(self, update: Update):
```

Ruff (0.12.2)

Undefined name `Update`

(F821)